cypress/support/pages/profile.pageObject.js

```diff
@@ -0,0 +1,16 @@
+import PageObject from '../PageObject';
+
+class profilePageObject extends PageObject {
+
+usernameValue(name) {
```

Comment:
Function names should be in camelCase and should be more descriptive. Rename 'usernameValue' to 'checkUsernameValue'

Reply:
done